roles/lnls-ans-role-qt/defaults/main.yml

```diff
@@ -12,10 +12,10 @@ qt_pip_packages_state: present
 qt_src_force_install: false
 
 # Qt source version when installing from source
-qt_src_version: 5.11.3
+qt_src_version: "{{ pkg_version_qt_src }}"
 
 # Qt script checksum. Extracted from website
-qt_src_checksum: md5:a9219194938e3e4ebe999a853a379a3f
+qt_src_checksum: md5:21c3b16f851697fa8da8009f73694373
```

Comment:
The checksum should also be a variable in my opinion, as it depends completely on the Qt package version.

Reply:
Yes, I completely agree. Qt provides a file with the checksum data:
[https://download.qt.io/archive/qt/5.13/5.13.1/md5sums.txt]()
However, since I don't have a lot of experience with ansible, it would take me too much time to make this change.
But, If you think it is better to fix this in this PR, feel free to do it @lerwys !

Reply:
done!